examples/finding_best_open_source_embedding_model/compose.yaml

```diff
@@ -0,0 +1,20 @@
+name: pgai-ollama
+services:
+  db:
+    image: timescale/timescaledb-ha:pg17
+    environment:
+      POSTGRES_PASSWORD: postgres
+      OPENAI_API_KEY: "sk-proj..."
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./data:/var/lib/postgresql/data
```

Comment:
The volume mount should actually look like this:
```
volumes:
         - data:/home/postgres/pgdata/data
```
And then 
```
volumes:
     data:
```

At the end of the file. This was an error in the previous quick start compose. Without this line though if the user shuts down their instance (e.g. because the y didn't add an API key from the start) they will lose all their data (also the created embeddings.

Reply:
I updated the `compose.yaml` file to reflect it!